paasta_tools/setup_prometheus_adapter_config.py

```diff
@@ -347,14 +348,15 @@ def create_instance_active_requests_scaling_rule(
 
 def create_instance_uwsgi_scaling_rule(
     service: str,
-    instance: str,
-    autoscaling_config: AutoscalingParamsDict,
+    instance_config: KubernetesDeploymentConfig,
     paasta_cluster: str,
-    namespace: str = "paasta",
 ) -> PrometheusAdapterRule:
     """
     Creates a Prometheus adapter rule config for a given service instance.
     """
+    autoscaling_config = instance_config.get_autoscaling_params()
+    instance = instance_config.instance
+    namespace = instance_config.get_namespace() or "paasta"
```

Comment:
"paasta" is already the default (and get_namespace should always return a non-empty string)
```suggestion
    namespace = instance_config.get_namespace()
```

Reply:
(we'll want to do this change here and in the other places we grab the namespace)

Reply:
Done.